components/page-actions.tsx

```diff
@@ -9,6 +9,7 @@ import NewProposal from "components/create-proposal";
 import CreatePullRequestModal from "components/create-pull-request-modal";
 import ForksAvatars from "components/forks-avatars";
 import GithubLink from "components/github-link";
+import KycModal from "components/modals/kyc-session";
```

Comment:
change the default name of this export, or change the name of the import: Its hard to find the component because the name of the component doesn't match the import name